files/jsondata/GroupData.json

```
@@ -765,7 +765,7 @@
     "HTML Sanitizer API": {
       "overview": ["HTML Sanitizer API"],
       "interfaces": ["Sanitizer"],
-      "methods": ["Sanitizer.sanitize", "Sanitizer.sanitizeToString"],
+      "methods": ["Sanitizer.sanitize()", "Sanitizer.sanitizeFor()"],
```

There is no need to add properties and methods of the interfaces listed under interfaces (that's only for methods that live in other interfaces).
```suggestion
      "methods": [],
```